.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java-version: [ '17', '18', '19', '20', '21', '22-ea' ]
```

Comment:
Probably want to wait until the actions/setup-java is updated with Java 22 GA support.

Reply:
Agreed